scipy/stats/_distr_params.py

```diff
@@ -97,6 +97,7 @@
     ['semicircular', ()],
     ['skewcauchy', (0.5,)],
     ['skewnorm', (4.0,)],
+    ['studentized_range', (3.0, 10.0)],
```

Comment:
I think we also need to add `studentized_range` to the `invdistcont` list with a parameter that is invalid.

Reply:
resolved